plugins/rs_lisp.py

```diff
@@ -0,0 +1,472 @@
+            "squared": "🄰🄱🄲🄳🄴🄵🄶🄷🄸🄹🄺🄻🄼🄰🄱🄲🄳🄴🄵🄶🄷🄸🄹🄺🄻🄼🄽🄾🄿🅀🅁🅂🅃🅄🅅🅆🅇🅈🅉🄽🄾🄿🅀🅁🅂🅃🅄🅅🅆🅇🅈🅉",
+            "squared_neg": "🅰🅱🅲🅳🅴🅵🅶🅷🅸🅹🅺🅻🅼🅰🅱🅲🅳🅴🅵🅶🅷🅸🅹🅺🅻🅼🅽🅾🅿🆀🆁🆂🆃🆄🆅🆆🆇🆈🆉🅽🅾🅿🆀🆁🆂🆃🆄🆅🆆🆇🆈🆉",
+            "subscript": "ₐBCDₑFGₕᵢⱼₖₗₘₐbcdₑfgₕᵢⱼₖₗₘₙₒₚQᵣₛₜᵤᵥWₓYZₙₒₚqᵣₛₜᵤᵥwₓyz",
+            "superscript": "ᴬᴮᶜᴰᴱᶠᴳᴴᴵᴶᴷᴸᴹᵃᵇᶜᵈᵉᶠᵍʰⁱʲᵏˡᵐᴺᴼᴾQᴿˢᵀᵁⱽᵂˣʸᶻⁿᵒᵖqʳˢᵗᵘᵛʷˣʸᶻ",
+            "inverted": "ɐqɔpǝɟƃɥıɾʞןɯɐqɔpǝɟƃɥıɾʞןɯuodbɹsʇn𐌡ʍxʎzuodbɹsʇnʌʍxʎz",
+            "reversed": "AdↃbƎꟻGHIJK⅃MAdↄbɘꟻgHijklmᴎOꟼpᴙꙄTUVWXYZᴎoqpᴙꙅTUvwxYz",
+            "smallcaps": "ABCDEFGHIJKLMᴀʙᴄᴅᴇꜰɢʜɪᴊᴋʟᴍNOPQRSTUVWXYZɴᴏᴩqʀꜱᴛᴜᴠᴡxyᴢ",
+            "weird1": "ልጌርዕቿቻኗዘጎጋጕረጠልጌርዕቿቻኗዘጎጋጕረጠክዐየዒዪነፕሁሀሠሸሃጊክዐየዒዪነፕሁሀሠሸሃጊ",
+            "weird2": "ДБҀↁЄFБНІЈЌLМаъсↁэfБЂіјкlмИФРQЯЅГЦVЩЖЧZиорqѓѕтцvшхЎz",
+            "weird3": "ค๒ƈɗﻉिﻭɦٱﻝᛕɭ๓ค๒ƈɗﻉिﻭɦٱﻝᛕɭ๓กѻρ۹ɼรՇપ۷ฝซץչกѻρ۹ɼรՇપ۷ฝซץչ",
+            "weird4": "αв¢∂єƒﻭнιנкℓмαв¢∂єƒﻭнιנкℓмησρ۹яѕтυνωχуչησρ۹яѕтυνωχуչ",
+            "weird5": "ค๒ς๔єŦﻮђเןкɭ๓ค๒ς๔єŦﻮђเןкɭ๓ภ๏קợгรՇยשฬאץչภ๏קợгรՇยשฬאץչ",
+            "weird6": "ﾑ乃cd乇ｷgんﾉﾌズﾚﾶﾑ乃cd乇ｷgんﾉﾌズﾚﾶ刀oｱq尺丂ｲu√wﾒﾘ乙刀oｱq尺丂ｲu√wﾒﾘ乙",
+            "sbancient": ""
+        }
+        if args[0].lower() == 'help':
+            return "```\nAVAILABLE TRANSCODINGS:\n" + "\n".join(alt_code.keys()) + "```"
+        elif args[0].lower() not in alt_code:
+            raise CustomCommandSyntaxError(f"{args[0].lower()} is not a supported transcoding. Use (transcode "
+                                           f"\"help\") to get a list of available transcodings.")
+        else:
+            alt_code = alt_code[args[0].lower()]
+    else:
+        def_code = args[0]
+        alt_code = args[1]
+        if len(def_code) != len(alt_code):
+            raise CustomCommandSyntaxError("transcode: To and From transcoding patterns must be the same length.")
+    return string.translate(str.maketrans(def_code, alt_code))
+
+
+def standard_env(*_, **kwargs):
+    env = Env(**kwargs)
+    env.update(vars(math))
+    env.update({
+        '+': op.add, '-': op.sub, '*': op.mul, '/': op.truediv, '//': op.floordiv, '%': op.mod,
+        '>': op.gt, '<': op.lt, '>=': op.ge, '<=': op.le, '==': op.eq, '<>': op.xor,
+        '!=': lambda *x: op.not_(op.eq(*x)),
+        '#': lambda x, y: y[x],
+        'abs': abs,
+        'append': lambda x, y: x.append(y) if type(x) == list else x + y,
+        'apply': lambda proc, args: proc(*args),
+        'do': lambda *x: x[-1],
+        'car': lambda x: x[0],
+        'cdr': lambda x: x[1:],
+        'cons': lambda x, y: [x] + y,
+        'is': op.is_,
+        'in': lambda x, y: x in y,
+        'len': len,
+        'list': lambda *x: list(x),
+        'l': lambda *x: list(x),
+        'range': range,
+        'list?': lambda x: isinstance(x, list),
+        'map': lambda *x: list(map(*x)),
+        'sum': sum,
+        'max': max,
+        'filter': filter,
+        'reduce': reduce,
+        'sort': sorted,
+        'reverse': lambda x: x[::-1],
```

Comment:
Could use built-in `reversed()` for this

Reply:
`reversed()` returns an iterator, which isn't terribly good for the current way RSLisp handles lists. `list(reversed())` would eliminate the performance gain.

And slice looks prettier.